### ISSUE TITLE
Icons in notifications are not labelled as decorative

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1139]
### Current behaviour
Icons in notifications UI are being focused when using screen readers. The alt-text is empty so it doesn't read anything.

### Expected behaviour
Icons should be labelled as decorative so screen readers ignore them.

### Devices
- iPhone SE iOS 16.3.1
- Oneplus 6 Anddroid 11.1.2.2

### Build
- 905 and above

**Comment by jleach** [comment]
I got this to work in settings by just disabling accessibility for the icons.
